bundles/org.openhab.binding.draytonwiser/README.md

```diff
@@ -0,0 +1,248 @@
+
+#### Boiler Controller
+
+| Channel                      | Item Type            | Description                                              |
+|------------------------------|----------------------|----------------------------------------------------------|
+| `heatingOverride`            | Switch               | State of the heating override button on the controller   |
+| `heatChannel1Demand`         | Number:Dimensionless | Current demand level of heating channel 1                |
+| `heatChannel1DemandState`    | Switch               | Is channel 1 calling the boiler for heat                 |
+| `heatChannel2Demand`         | Number:Dimensionless | Current demand level of heating channel 2                |
+| `heatChannel2DemandState`    | Switch               | Is channel 2 calling the boiler for heat                 |
+| `currentSignalRSSI`          | Number               | Relative Signal Strength Indicator                       |
+| `currentWiserSignalStrength` | String               | Human readable signal strength                           |
+| `currentSignalStrength`      | Number               | Signal strength value that maps to qualityofservice icon |
+
+#### Hot Water
+
+| Channel                  | Item Type    | Description                                              |
+|--------------------------|--------------|----------------------------------------------------------|
+| `hotWaterOverride`       | Switch       | State of the hot water override button on the controller |
+| `hotWaterDemandState`    | Switch       | Is hot water calling the boiler for heat                 |
+| `hotWaterBoosted`        | Switch       | Is hot water currently being boosted                     |
+| `hotWaterBoostRemaining` | Number:Time  | How long until the boost deactivates in minutes          |
+
+#### Room
+
+| Channel              | Item Type            | Description                                                                  |
+|----------------------|----------------------|------------------------------------------------------------------------------|
+| `currentTemperature` | Number:Temperature   | Currently reported temperature                                               |
+| `currentHumidity`    | Number:Dimensionless | Currently reported humidity (if there is a room stat configured in this room |
+| `currentDemand`      | Number:Dimensionless | Current heat demand percentage of the room                                   |
+| `heatRequest`        | Switch               | Is the room actively requesting heat from the controller                     |
+| `roomBoosted`        | Switch               | Is the room currently being boosted                                          |
+| `roomBoostRemaining` | Number:Time          | How long until the boost deactivates in minutes                              |
+| `windowState`        | Switch               | Is the window open or closed?                                                |
```

Comment:
If I see correctly you used `Contact` in the code.

Reply:
Yes, I'll fix that in the Readme

Reply:
Fixed in 9a6a1e1